plugins/examples/script-expert-scoring/src/yamlRestTest/resources/rest-api-spec/test/script_expert_scoring/20_score.yml

```diff
@@ -4,33 +4,71 @@
 setup:
   - do:
       indices.create:
-          index:  test
+        index: test
 
   - do:
       index:
-        index:  test
-        id:     "1"
-        body:   { "important_field": "foo" }
+        index: test
+        id: "1"
+        body: { "important_field": "foo" }
   - do:
-        index:
-          index:  test
-          id:     "2"
-          body:   { "important_field": "foo foo foo" }
+      index:
+        index: test
+        id: "2"
+        body: { "important_field": "foo foo foo" }
   - do:
-        index:
-          index:  test
-          id:     "3"
-          body:   { "important_field": "foo foo" }
+      index:
+        index: test
+        id: "3"
+        body: { "important_field": "foo foo" }
 
   - do:
-      indices.refresh: {}
+      indices.refresh: { }
+
 ---
 "document scoring":
+
+  - skip:
+      cluster_features: [ "gte_v8.16.0" ]
+      reason: "bug fixed where explanations were throwing npe prior to 8.16"
+
```

Comment:
```suggestion
```

Since this is just the same test as it was before, no need to block it from running